cmd/state-tray/internal/open/open_lin.go

```diff
@@ -28,3 +28,38 @@ func Prompt(command string) error {
 
 	return nil
 }
+
+func preferredShellWithFallback(fallback string) (string, error) {
+	currentUser, err := user.Current()
+	if err != nil {
+		return "", locale.WrapError(err, "err_user_unknown", "Cannot get current user")
```

Comment:
So the fallback is only used if we make it through to reading the /etc/passwd? We should use it for any of these errors.

Suggest making this function just error out, and if it errors, log it from the consumer and fall back to bash from the consumer.

Reply:
This package does no other logging, and I think it's not worth adding that. I suggest that we just use the fallback and don't worry about it at this time.

Reply:
I'd like to have the logging so we can gauge whether this is actually a sensible mechanic to use.

Reply:
Does this mean logging to Rollbar in order for us to see it?